test/unit/InputFileResolverSpec.ts

```diff
@@ -53,7 +53,7 @@ describe('InputFileResolver', () => {
     let result: Error;
     beforeEach(() => {
       try {
-        sut = new InputFileResolver(undefined, [<any>{ included: false, mutated: true }]);
+        sut = new InputFileResolver([''], [<any>{ included: false, mutated: true }]);
```

Comment:
I think an empty array is cleaner here. That stays a bit more true to the purpose.